modules/service/src/main/scala/lucuma/odb/graphql/mapping/StepRecordMapping.scala

```diff
@@ -53,9 +63,7 @@ trait StepRecordMapping[F[_]] extends StepRecordView[F]
         EffectField("interval",  intervalHandler, List("id")),
         SqlObject("stepConfig"),
         SqlField("observeClass", StepRecordView.ObserveClass),
-
-        // TBD: stepQaState
-
+        EffectField("qaState", qaStateHandler, List("id")),
```

Comment:
I think it likely possible to do this without resorting to an `EffectField` but was stymied by the fact that the data it needs is not found in a simple `Leaf`, like for example instances where a `CirceFieldJson` is used.  Also, the underlying query is trivial, though the `EffectHandler` trappings are convoluted.